src/site/content/en/blog/vitals-tools/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: "Tools to measure Core Web Vitals"
+subhead: Your favorite developer tools can now measure the Core Web Vitals metrics.
+authors:
+  - addyosmani
+  - egsweeny
+date: 2020-05-28
+description: |
+  Read about the newly announced Core Web Vitals measurement support across popular web developer tools like Lighthouse, PageSpeed Insights, Chrome UX Report, and many others.
+hero: hero.png
```

Comment:
I accidentally optimized this but I checked it on staging and it seems fine. If it ends up looking grainy we can just re-upload a higher fidelity version.

P.S. what's up with the 3 straight lines on the DevTools logo 😆 

![image](https://user-images.githubusercontent.com/4713486/83095264-c66f4000-a057-11ea-940b-31ff26841165.png)

Reply:
I'll correct this in my next push. Something funky happened with the SVG import first time around :)